README.md

```
@@ -267,6 +269,22 @@ Install Kr2FA using one of the following methods:
    ```
 
+###  Configuration
+
+Edit `plugins/Velocity2FA/config.json` after first run. New option:
+
+- `messagePrefix`: A short tag prepended to all player/console messages sent by the plugin. Default: `[Kr2FA]`.
+
+Example snippet:
+```json
+{
+	"serverName": "Proxy-01",
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)